cmd/runtimetest/main.go

```diff
@@ -244,6 +245,45 @@ func validateROPaths(spec *rspec.Spec) error {
 	return nil
 }
 
+func getThisCgroupPath(subsystem string) (string, error) {
+	f, err := os.Open("/proc/self/cgroup")
+	if err != nil {
+		return "", err
+	}
+	defer f.Close()
+
+	s := bufio.NewScanner(f)
+	for s.Scan() {
+		if err := s.Err(); err != nil {
+			return "", err
+		}
+
+		text := s.Text()
+		parts := strings.Split(text, ":")
+
+		for _, subs := range strings.Split(parts[1], ",") {
+			if subs == subsystem {
+				return parts[2], nil
+			}
+		}
+	}
+	return "", fmt.Errorf("cgroup %v not found", subsystem)
+}
+
+func validateCgroupsPath(spec *rspec.Spec) error {
+	fmt.Println("validating cgroupsPath")
+	expectedPath := spec.Linux.CgroupsPath
+	actualPath, err := getThisCgroupPath("devices")
```

Comment:
I think this needs some spec work before we can implement something reliable.  I expect we'll want to make this comparison for unified cgroups if the container process belongs to the unified hierarchy, and with ([some][1]?) v1 cgroups if the container does not belong to the unified hierarchy.

[1]: https://github.com/opencontainers/runc/issues/861